CMPUT301Project/app/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -13,4 +13,8 @@
         android:id="@+id/profile"
         android:title="Profile"
         android:icon="@drawable/ic_person_foreground"/>
+    <item
+        android:id="@+id/friends"
+        android:title="Friends"
+        android:icon="@drawable/ic_person_foreground"/>
```

Comment:
We should have a different icon here. Both profile and friends are ic_person_foreground

Reply:
I temporarily used this icon and forgot to change it. We can change it in another update

Reply:
I will look into this.